core/src/main/java/com/linecorp/armeria/server/AnnotatedHttpServiceMethod.java

```diff
@@ -626,6 +653,8 @@ static Parameter ofRequestObject(Class<?> type,
         private final String name;
         private final String defaultValue;
         private final RequestConverterFunction requestConverterFunction;
+        private final Map<String, ? extends Enum<?>> upperCaseEnum;
+        private final boolean isCaseSensitive;
```

Comment:
@hyangtack  I added this flag for the case when the element of the enum has same spell but different case.

Reply:
Could we use `upperCaseEnum` variable as a flag instead of `isCaseSensitive`? We may use `upperCaseEnum` only if it's not `null`. Also, it'd be better to make `upperCaseEnum` as `final`. :-)